packages/service/src/domain/commands/processors/queue-command.ts

```diff
@@ -6,12 +6,14 @@ import CommandDefinition from '../model/command-definition';
 import { AddSongToQueueEvent } from '../../../event-stream/model/events';
 
 async function queueCommandProcessor(command: Command): Promise<CommandProcessingResponse> {
-  const songName = command.rawArgs;
+  const commandArgs = command.getArgsByDelimiter(' ');
+  const atTheBeginning = (commandArgs.length > 1 && commandArgs[0] === '-n');
```

Comment:
Implementacja spoko, warto jeszcze na devkę wdrożyć i przetestować.

Cały czas się zastanawiam nad tą przełączką `-n`. Wydaje mi się że lepsze byłoby `--next` i liczyłem, że się reszta wypowie. Może trzeba po prostu na Slacku zagadać :P

Reply:
Nie widziałem tych komentarzy, nigdzie nie dostałem powiadomień.
Mi `-n` nie przeszkadza (mogłoby być nawet -f bo forceujemy na początek kolejki ;p)
Nie ma co wydłużać komend

Reply:
Zdziwiłem się trochę widząc na kanale "/fm q -n <fraza>", musiałem spojrzeć do helpa. "--next" byłby perfekcyjnie zrozumiały, choć rozumiem argument o nie przedłużaniu komend.
Chociaż zaraz, to jest komenda, której powinniśmy rzadko używać, więc --next mi się podoba :) --force niezbyt, bo nie jest wiadome co wymuszamy

Reply:
to może `--head` `--tail`, gdzie `tail` jest domyślne

Reply:
można pójść na konsensus i dodać zarówno dla `-n` jak i `--next` ;)

Reply:
Ja bym był za `--next` po prostu.
1. Różnica długości między `-n`, a `--next` nie jest duża
2. To nie jest komenda do używania często
3. `--next` jest bardziej opisowe
4. Robienie i `-n` i `--next` wprowadza tylko niepotrzebną złożoność do kodu

Reply:
poprawione na --next